Builds/linux/README.md

```diff
@@ -107,6 +107,10 @@ followed by:
 cmake -DCMAKE_BUILD_TYPE=Debug ..
 ```
 
+If your operating system does not provide static libraries (Arch Linux, and 
```

Comment:
Arch and Manjaro are closely related. Maybe mention Arch, Alpine and OpenSUSE instead? I'm not sure about Red Hat based distros.

Reply:
Are you sure Alpine doesn't [provide static libs](https://pkgs.alpinelinux.org/package/edge/main/x86_64/openssl-libs-static) for the necessary stuff?

I'm not very knowledgeable about OpenSUSE, but it _does_ seem like they have [a policy against packaging static libs](https://en.opensuse.org/openSUSE:Shared_library_packaging_policy#Best_Practices), so maybe that can be added.

Reply:
I think it'd be great to add additional details here, but I'd like to merge this. Perhaps we can further enhance this document in a follow-up?